x-pack/plugins/cross_cluster_replication/public/app/components/remote_clusters_form_field.js

```diff
@@ -152,10 +151,9 @@ export class RemoteClustersFormField extends PureComponent {
           {' '}
           {/* Break out of EuiFormRow's flexbox layout */}
           <EuiButtonEmpty
-            {...routing.getRouterLinkProps(
+            href={routing.getHrefToRemoteClusters(
```

Comment:
Why not use `reactRouterNavigate` here? `getRouterLinkProps` was providing `href` and `onClick` originally.

Reply:
Because in this case, we navigate to another section. We can not use history.push since push method  navigate relative to one section

Reply:
@mattkime @cjcenizal @joshdover 
Today we found one more way how to achieve redirect without hard page reload between sections. We can use `reactRouterNavigate` but instead of history use `history.parentHistory`. In this case, the url should be relative to the part: `app/management`

Not sure that it's a blocker for merging this PR, all teams can update links in future themselves